src/cpp/r/R/Tools.R

```diff
@@ -27,6 +27,12 @@ assign(".rs.toolsEnv", function()
    .rs.Env
 }, envir = .rs.Env)
 
+# target environment for symbol lookup, with all necessary base packages
+assign(".rs.symbolLookupEnv", function()
+{
+   new.env(parent = .rs.toolsEnv())
```

Comment:
Not clear to me whether this should inherit from .rs.toolsEnv or just the base packages. I don't think it really needs .rs.toolsEnv, but I'm not sure if there's harm in including it either. 

Also there may be a better name for this environment. `.rs.apiEnv` was another consideration, but I thought it might be confusing to have it start with the `.rs.api` prefix